test/verify/check-machines-create

```diff
@@ -689,7 +689,7 @@ class TestMachinesCreate(VirtualMachinesCase):
                 query_result = '{0}'.format(self.os_name)
                 # throws exception if grep fails
                 self.machine.execute(
-                    "osinfo-query os --fields=name | tail -n +3 | sed -e 's/\s*|\s*/|/g; s/^\s*//g; s/\s*$//g' | grep '{0}'".format(
+                    "osinfo-query os --fields=name | tail -n +3 | sed -e 's/\\s*|\\s*/|/g; s/^\\s*//g; s/\\s*$//g' | grep '{0}'".format(
```

Comment:
Let's also use a raw string here instead of these unsightly `\\`.

Reply:
I have an irrational tendency to use raw strings only for regexps, (I used to think the "r" stood for regexp :rofl:) but there's actually no reason not to use them here as well...